18F/identity-idp
# LG-12492 Update getting started screen for legacy selfies

## 🎫 Ticket

Link to the relevant ticket:
[LG-12492](https://cm-jira.usa.gov/browse/LG-12942)

## 🛠 Summary of changes

Changed the welcome screen text for users who are undergoing step-up IdV (they were verified without a selfie, but now must be re-verified with a selfie).

## 📜 Testing Plan

- Create a user and proceed through IdV.
- Verify that the new text does not appear on the getting started
  screen
- Stop the app and enable selfies (Change `NullServiceProviderSession#selfie_required?` to return true)
- Restart the app and re-enter IdV
- Verify that the new text appears on the getting started screen.




## 👀 Screenshots

If relevant, include a screenshot or screen capture of the changes.

<details>
<summary>Beginning initial IdV:</summary>

<img width="623" alt="Screenshot 2024-03-05 at 2 24 39 PM" src="https://github.com/18F/identity-idp/assets/101212334/1ca8f369-730b-47a5-a0ef-f826fe479057">

</details>

<details>
<summary>Beginning initial IdV (French):</summary>

![Screenshot 2024-03-11 at 10 19 45 AM](https://github.com/18F/identity-idp/assets/101212334/dc72e869-dade-4021-8068-b6bcddb73523)

</details>

<details>
<summary>Beginning initial IdV (Spanish):</summary>

![Screenshot 2024-03-11 at 10 20 24 AM](https://github.com/18F/identity-idp/assets/101212334/57fb28db-c0af-4bde-bacb-3a0977b3c8bd)

</details>

<details>
<summary>Beginning step-up IdV:</summary>
<img width="621" alt="Screenshot 2024-03-05 at 2 26 00 PM" src="https://github.com/18F/identity-idp/assets/101212334/9f814a86-37ec-4dd0-95a3-4f40fe748236">

</details>
<details>
<summary>Beginning step-up IdV (French):</summary>

![Screenshot 2024-03-11 at 10 38 46 AM](https://github.com/18F/identity-idp/assets/101212334/e9263312-9b79-4126-813b-4a45bbb27b35)

</details>
<details>
<summary>Beginning step-up IdV (Spanish):</summary>

![Screenshot 2024-03-11 at 10 39 12 AM](https://github.com/18F/identity-idp/assets/101212334/b9ca0637-f16b-463f-a391-bfc20e6dcab0)

</details>
